packages/shared/src/components/RecommendedTags.tsx

```diff
@@ -5,7 +5,7 @@ import type { Tag } from '../graphql/feedSettings';
 
 interface RecommendedTagsProps {
   isLoading: boolean;
-  tags: Tag[];
+  tags?: Tag[];
```

Comment:
now these optional things are starting to scare me 😅

Reply:
Its actually better since when we turn on strict ts option it will scream if you try to access something that can be undefined. Only thing.. a lot of work to fix all the types.. 😆